src/components/FeatureViews/Accounts/TabComponents/EnrollmentPayment.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Loading from "components/OmouComponents/Loading";
+import PaymentTable from "./PaymentTable";
+import {useQuery} from "@apollo/react-hooks";
+import Typography from "@material-ui/core/Typography";
+import gql from "graphql-tag";
+
+export const GET_ENROLLMENT_PAYMENTS = gql`
+query EnrollmentPayments($enrollmentId: ID!) {
+  enrollment(enrollmentId: $enrollmentId) {
+    paymentList {
+      __typename
```

Comment:
You don't need to pass ```__typename``` if you aren't using it. They automatically populate after you make a graphql request.

Reply:
Ok Great, made that update